test/rules/jsx-self-close/test.lint.fix

```diff
@@ -0,0 +1,27 @@
+<div>
+    Contents
+</div>
+
+
+
+<span/>
+
+<button />
+
+
+
+<h2 class="colouring" contents="B"/>
```

Comment:
let's include a space right before the `/` here, to match what prettier would do with formatting

Reply:
Thanks for the review, I added the space for all the self-closing tags. Is it ok ?